NEWS.md

```diff
@@ -247,6 +247,8 @@ Library improvements
 
   * A new `Dates.Time` type was added that supports representing the time of day with up to nanosecond resolution ([#12274]).
 
+  * New `@macroexpand` macro as convenient alternative to the `macroexpand` function ([#18660]).
```

Comment:
Should probably be "as a" rather than just "as"

Also good to add `[ci skip]` to commits that only touch NEWS.md.